modules/gha/workflow.go

```diff
@@ -180,7 +180,7 @@ func (gha *Gha) Workflow(
 		w = w.onPullRequest(nil, onPullRequestBranches, nil)
 	}
 	if onPullRequestPaths != nil {
-		w = w.onPullRequest([]string{"paths"}, nil, onPullRequestPaths)
+		w = w.onPullRequest(nil, nil, onPullRequestPaths)
```

Comment:
`paths` isn't actually a valid event. See https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request.

Reply:
This is valid: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request

<img width="947" alt="image" src="https://github.com/user-attachments/assets/13d2f5b4-8947-4c5b-a7e0-45a3afa11163">

Reply:
Sorry, it's valid as a filter yes, but not valid as a `types` value.